libraries/python/src/internal/portfolio.py

```
@@ -0,0 +1,97 @@
+    side_count = total_positions_count // 2
+
+    if long_count != side_count:
+        message = f"Expected {side_count} long side positions, found: {long_count}"
+        raise ValueError(message)
+
+    if short_count != side_count:
+        message = f"Expected {side_count} short side positions, found: {short_count}"
+        raise ValueError(message)
+
+    if data.lazyframe.collect().height != total_positions_count:
+        message = f"Expected {total_positions_count} total positions, found: {data.lazyframe.collect().height}"  # noqa: E501
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)